### PR TITLE
Kupniewski add opening window

### PR DESCRIPTION
**Pull request by lormniczek**
Poprawiłem błędy z nullami i dodałem nazwy do propertiesów, działa we wszystkich językach od violeta.

### CODE REVIEW

**Review by abizukojc — CHANGES_REQUESTED** [review]
You have conflicts. Fix it.

File: violetproduct-swing/src/main/java/com/horstmann/violet/application/gui/SplashFrame.java [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * This desktop frame contains panes that show menu to create/open projects.
+ *
+ * @author Maciej Kupniewski
```
Comment (abizukojc) [inline violetproduct-swing/src/main/java/com/horstmann/violet/application/gui/SplashFrame.java]: Remove this

File: violetproduct-swing/src/main/java/com/horstmann/violet/application/gui/SplashFrame_fr.properties [hunk 2]
```diff
@@ -0,0 +1,12 @@
+dialog.open_file_failed.text=Impossible d'ouvrir le fichier
+workspace.unsaved_prefix=Unsaved
+nameBtnNewProject=Nouveau projet
+nameBtnOpenProject=Ouvrir un projet
+nameBtnOpenRecent=Le dernier projet
+nameBtnActivityDiagram=Diagramme d'activit�
```
Comment (abizukojc) [inline violetproduct-swing/src/main/java/com/horstmann/violet/application/gui/SplashFrame_fr.properties]: Use unicode here

**Review by abizukojc — CHANGES_REQUESTED** [review]

File: violetproduct-swing/src/main/java/com/horstmann/violet/application/gui/SplashFrame.java [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+    private List<IDiagramPlugin> diagramPlugins;
+
+    /**
+     * Constructs frame with 3 buttons to create/open or open recent project.
+     *
+     * @param mainFrame it's for add workspace to main frame.
+     */
+    public SplashFrame(final MainFrame mainFrame) {
+        mainPanel.setLayout(cardLayout);
+        frame.setTitle("Okno Powitalne");
```
Comment (bartoszsledz) [inline violetproduct-swing/src/main/java/com/horstmann/violet/application/gui/SplashFrame.java]: use properties file.. pl,eng,fr,ger

File: violetproduct-swing/src/main/java/com/horstmann/violet/application/gui/SplashFrame_fr.properties (hunk 2 above)
Comment (bartoszsledz) [inline violetproduct-swing/src/main/java/com/horstmann/violet/application/gui/SplashFrame_fr.properties]: /u

File: violetproduct-swing/src/main/java/com/horstmann/violet/application/gui/SplashFrame.java [hunk 4]
```diff
@@ -0,0 +1,329 @@
...
+        btnOpenRecent = new JButton(btnNameOpenRecent);
+
+        btnActivityDiagram = new JButton(btnNameActivityDiagram);
+        btnClassDiagram = new JButton(btnNameClassDiagram);
+        btnComunicationDiagram = new JButton(btnNameComunicationDiagram);
+        btnSequenceDiagram = new JButton(btnNameSequenceDiagram);
+        btnStateDiagram = new JButton(btnNameStateDiagram);
+        btnUseCaseDiagram = new JButton(btnNameUseCaseDiagram);
+        btnObjectDiagram = new JButton(btnNameObjectDiagram);
+
```
Comment (bartoszsledz) [inline violetproduct-swing/src/main/java/com/horstmann/violet/application/gui/SplashFrame.java]: this not working with feature 'settings language in project'.. always shows in polish, try fix it

Comment (bartoszsledz) [inline violetproduct-swing/src/main/java/com/horstmann/violet/application/gui/SplashFrame.java]: this not working with feature 'settings language in project'.. always shows in polish, try fix it

**Review by orchowski — CHANGES_REQUESTED** [review]
Resolve conflicts

**Review by abizukojc — CHANGES_REQUESTED** [review]
It's ok, but resolve conflicts first.

**Review by abizukojc — APPROVED** [review]

**Review by orchowski — APPROVED** [review]

**Pull request merged**